dev-tools/audius-cmd

```diff
@@ -4,6 +4,8 @@ set -e
 cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"/compose  # audius-protocol/dev-tools/compose
 export COMPOSE_PROJECT_NAME='audius-protocol'
 
+turbo run build --filter=@audius/commands
```

Comment:
This is funny. If you run it _inside_ the container with docker compose, it takes 5-6 seconds. If you run it _outside_ the container in your normal turbo environment (where you have a daemon and remote caching), it takes half a second....

Reply:
docker kinda sucks, esp on macos :(